### DISCUSSION TITLE
Is the unstable_createRemixStub testing helper officially supported?

### DISCUSSION THREAD

**Discussion by TheWarBadger** [remix-run/remix#7564] [General]
Hi,

I need to use a `<Link>` element in a unit test, and came up against https://github.com/remix-run/remix/issues/4506.

This is closed due to being tracked in https://github.com/remix-run/remix/issues/4488.

4488 has been closed as released, however the result still has the `unstable` prefix and the only documentation I've found is example unit tests (https://github.com/remix-run/remix/blob/dev/packages/remix-testing/__tests__/stub-test.tsx) linked to at the top of the community repository (https://github.com/jrestall/remix-stubs).

Any idea what the status is? Is there a plan to support this? Would it be better to use the community versions?

**Comment by kiliman** [comment]
`unstable` doesn't mean _unofficial_. It simply means that the API may change in the future. The team wants people to use it to see what, if any, changes are needed.

Kent is using it in his Epic Stack and also teaches it in his workshop. So I think it's safe to use.
